resources/js/components/loginBox2.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="">
+    <div class="loginBox2"></div>
```

Comment:
Is there a reason we are having two login components? I noticed `loginBox.vue` and `loginBox2.vue`

Reply:
the loginBox.vue is for the login box: where we have our textbox etc, whle the other one is for the other box from the UI, i dont know what to call it